src/utils/format.js

```diff
@@ -278,3 +278,33 @@ export function formatLiquidityPoolShare(value: ?BigNumber | string) {
 
   return formatPercentValue(value, { decimalPlaces: 4, stripTrailingZeros: true });
 }
+
+export const formatAmountDisplay = (
+  amountRaw: string | number,
+  leftSymbol?: string,
+  minimumFractionDigits?: number,
+): string => {
+  const amount = typeof amountRaw === 'number' ? `${amountRaw}` : amountRaw;
+
+  // check string to avoid underflow
```

Comment:
🙇

Reply:
oh lol, i just copied that from the buidler